### PR TITLE
Release 2021.11.3

### PR DESCRIPTION
**Pull request by brentschaus-okta**
## Description:
- **What's changed?** Adding release notes and docs for 2021.11.3
- **Is this PR related to a Monolith release?** Yes, 2021.11.3

### Resolves:
* [OKTA-448198](https://oktainc.atlassian.net/browse/OKTA-448198)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
-### Weekly Release 2021.11.2
+### Weekly release 2021.11.3
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bug fixed in 2021.11.3](#bug-fixed-in-2021-11-3) | December 02, 2021 |
+
+#### Bug fixed in 2021.11.3
+
+When the [ThreatInsight configuration](/docs/reference/api/threat-insight/#threatinsight-configuration-api-operations) endpoint was called, the response included non-standard date and timestamp formats for the `created` and `lastUpdated` keys. (OKTA-445365)
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:15]:
```suggestion
When the [ThreatInsight configuration](/docs/reference/api/threat-insight/#threatinsight-configuration-api-operations) endpoint was called, the response included non-standard date and timestamp formats for the `created` and `lastUpdated` properties. (OKTA-445365)
```

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 2]
```diff
@@ -257,7 +267,7 @@ The Okta [Custom Administrator Roles](/docs/reference/api/roles/index.md) API pr
 
 #### Okta Sign-In Widget, version 5.9.1
 
-For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.9.1). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-416419-->
+For details about this release, see the Okta [Sign-In Widget release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.9.1). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-416419-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:270]: Should we put "Notes" in lower case too? (i.e. "Sign-In Widget release notes") Also, should we use "Sign-In Widget guide"?

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 3]
```diff
@@ -426,7 +436,7 @@ When the [Features endpoint of the Apps API](/docs/reference/api/apps/#update-fe
 
 #### Okta Sign-In Widget, version 5.7.0
 
-For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.7.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-391046-->
+For details about this release, see the Okta [Sign-In Widget release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.7.0). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-391046-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:439]: same question as before.. why do we lower case "release", but not "Notes"?

File: packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md [hunk 4]
```diff
@@ -1,12 +1,27 @@
...
+### Weekly release 2021.11.3
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Support for multiple device management statuses on a single device is now GA in Preview](#support-for-multiple-device-management-status-on-a-single-device-is-now-ga-in-preview) | December 02, 2021 |
+| [Bug fixed in 2021.11.3](#bug-fixed-in-2021-11-3) | December 02, 2021 |
+
+#### Support for multiple device management statuses on a single device is now GA in Preview
+
+Multiple users signed in to a single device now retain their own device management status. Users with a `managed` device status don't share that status with users who have an `unmanaged` device status. See [List devices (expand=user)](/docs/reference/api/devices/#usage-example-expand-user).<!--OKTA-447458-->
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md:18]: The second line sounds like the users are the actors "sharing" their device status with other users. From the example, it looks like the response from the `expand-user` option provides both `managed` and `unmanaged` status for each user? This is just a question
The PM approved, so this blurb should be ok.

**Review by vanngo-okta — APPROVED** [review]
I just have a few questions. The rest looks good, so I'm not holding you back. Approving...

File: packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md (hunk 4 above)
Comment (brentschaus-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021-okta-identity-engine/index.md:18]:
```suggestion
Multiple users signed in to a single device now retain their own device management status. Users with a `managed` device status don't share that status with users who have an `unmanaged` device status.<!--OKTA-447458-->
```

**Pull request merged**